runtime/zeitgeist/src/lib.rs

```diff
@@ -142,6 +145,13 @@ impl Contains<RuntimeCall> for IsCallable {
                     _ => true,
                 }
             }
+            // Permissioned contracts: Only deployable via utility.dispatch_as(...)
+            RuntimeCall::Contracts(inner_call) => match inner_call {
+                instantiate_with_code { .. } => false,
+                instantiate_with_code_old_weight { .. } => false,
+                upload_code { .. } => false,
+                _ => true,
+            },
```

Comment:
Will later look into this more specific to find potential calls that are not covered.